targets/nrf5x/bluetooth.c

```diff
@@ -3100,6 +3100,8 @@ JsVar *jsble_get_security_status(uint16_t conn_handle) {
     jsvObjectSetChildAndUnLock(result, "encrypted", jsvNewFromBool(status.encrypted));
     jsvObjectSetChildAndUnLock(result, "mitm_protected", jsvNewFromBool(status.mitm_protected));
     jsvObjectSetChildAndUnLock(result, "bonded", jsvNewFromBool(status.bonded));
+    bool isAdvertising = bleStatus & BLE_IS_ADVERTISING;
```

Comment:
Or could this be this, to be more accurate?

```suggestion
    bool isAdvertising = bleStatus & (BLE_IS_ADVERTISING | BLE_IS_ADVERTISING_MULTIPLE);
```

(also, happy to remove the intermediate variable if preferred)

Reply:
Great, thanks again!